test/unit/submission_collector_test.rb

```diff
@@ -211,14 +212,15 @@ def setup_collector
       setup do
         setup_collector
         @submission_collector.stubs(:start_collection_process)
-        #puts @groupings.slice(0..4).inspect
-        @submission_collector.push_groupings_to_queue(@groupings.slice(0..4))
+        @submission_collector.push_groupings_to_queue(@groupings.slice(0..2))
+        puts @regular_queue.all.inspect
+
       end
 
       should 'return the first grouping of the regular queue' do
-        assert_equal @groupings[0],
+        assert_includes @groupings[0..2],
           @submission_collector.get_next_grouping_for_collection
-        assert_equal @groupings.slice(0..4), @regular_queue.sort { |x,y| x.id <=> y.id}
+        assert_equal @groupings.slice(0..2), @regular_queue.sort { |x,y| x.id <=> y.id}
```

Comment:
Line is too long. [87/80]<br>Space missing after comma.<br>Space missing inside }.